src/Kmd.Logic.Gateway.Automation.Tool/CommonOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CommandLine;
+
+namespace Kmd.Logic.Gateway.Automation.Tool
+{
+    internal class CommonOptions
```

Comment:
Can we give different name then common

Reply:
I agree with @Araballi - what about `GatewayCommandOptions`?

Reply:
My suggestion is to name it just `BaseCommandOptions`. We already know that it is Gateway tool.

Reply:
Check now.

Reply:
LGTM